src/docs/contributing/platforms/index.mdx

```diff
@@ -110,8 +110,6 @@ If you wanted to make hide the content for a platform by default, but enable it
 
 ```markdown
 ---
-supported:
-  - native.ue4
```

Comment:
For this text to make sense, there still has to be a guide in the `supported` part of the example. It just can't be Unreal since Unreal is now a top-level platform. It has to be one of the frameworks under Native. It doesn't matter which one, but there has to be one there. I had chosen `wasm`, but you can use any of them.

Reply:
ok, thanks! For now I'll add wasm. I probably should discuss with Bruno what will fit here the best.